app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -75,7 +76,7 @@ public void testAddBlock() throws Exception {
     public void testAddBlock2() throws Exception {
         String newOwner = owner+"2";
         Block newBlock = BlockFactory.getBlock(TYPE_BLOCK, newOwner, sequenceNumber, ownHash,
```

Comment:
You should not be able to give a sequencenumber to the constructor upon creating a block